tests/unit/lms/views/api/blackboard/_schemas_test.py

```diff
@@ -21,11 +21,19 @@ def test_valid(self, list_files_response):
                 "id": "blackboard://content-resource/_7851_0/",
                 "updated_at": "2008-05-06T07:26:35.000z",
                 "display_name": "File_0.pdf",
+                "type": "File",
             },
             {
                 "id": "blackboard://content-resource/_7851_1/",
                 "updated_at": "1983-05-26T02:37:23.000z",
                 "display_name": "File_1.pdf",
+                "type": "File",
+            },
+            {
+                "id": "blackboard://content-resource/_7851_2/",
```

Comment:
Known issue: I don't think we want to wrap _folder_ IDs in `blackboard://` URLs. Folder IDs aren't going to get stored in our DB so this isn't necessary, we will only have to unwrap them again.